docs/pages/setup/reference/backends.mdx

```diff
@@ -216,15 +216,15 @@ running on an EC2 instance with an IAM role.
 You can add optional query parameters to the S3 URL. The Teleport Auth
 Service reads these parameters to configure its interactions with S3:
 
-`s3://bucket/path?region=us-east-1&endpoint=mys3.example.com&insecure=false&disablesse=false&acl=private`
+`s3://bucket/path?region=us-east-1&endpoint=mys3.example.com&insecure=false&disablesse=false&acl=private&use_fips_endpoint=true`
 
 - `region=us-east-1` - set the Amazon region to use.
 - `endpoint=mys3.example.com` - connect to a custom S3 endpoint.
 - `insecure=true` - set to `true` or `false`. If `true`, TLS will be disabled.
 - `disablesse=true` - set to `true` or `false`. If `true`, S3 server-side encryption will be disabled. If `false`, aws:kms (Key Management Service) will be used for server-side encryption. Other SSE types are not supported at this time.
 - `sse_kms_key=kms_key_id` - If set to a valid AWS KMS CMK key ID all objects uploaded to S3 will be encrypted with this key. Details can be found below.
 - `acl=private` - set the [canned ACL](https://docs.aws.amazon.com/AmazonS3/latest/userguide/acl-overview.html#canned-acl) to use. Must be one of the predefined ACL values.
-
+- `use_fips_endpoint=true` -  set to `true` or `false`. If `true`, [FIPS](https://aws.amazon.com/compliance/fips/) S3 endpoints will be used. If `false`, normal S3 endpoints will be used. If unset, the AWS Environment Variable `AWS_USE_FIPS_ENDPOINT` will determine which endpoint is used. In Teleport 10 or greater, FIPS endpoints will be used if Teleport is run with the `--fips` flag.
```

Comment:
Instead of saying "In Teleport 10 or greater", this should depend on the version of the documentation that's being shown. So in branch/v10, this should just say that "FIPS endpoints will be used if Teleport is run with --fips flag". In branch/v9, I'd say what the default behavior is.